pycytominer/cyto_utils/features.py

```diff
@@ -166,7 +169,7 @@ def drop_outlier_features(
 
     # Subset dataframe
     if samples != "all":
-        population_df = population_df.loc[samples, :]
+        population_df = population_df.query(samples)
```

Comment:
Would it make sense to use `population_df.query(...,inplace=True)` here to help conserve memory usage (avoiding the production of a copied dataframe)?

Reply:
I will make this change in the next commit! Thanks! (I hadn't realized `inplace` conserves memory, and historically, I haven't liked using `inplace` in pandas, probably for the wrong reasons)

Reply:
added in d9b06fe46f6f3b91bad9541224e7d67f6a1b9326